examples/java/src/main/java/glide/examples/ClusterExample.java

```diff
@@ -35,7 +35,7 @@ public static GlideClusterClient createClient() throws CancellationException, Ex
         // GLIDE is able to detect all cluster nodes and connect to them automatically
         // even if only one of them was configured
 
-        // Check GlideClusterClientConfiguration for additional options.
+        // Check <code>GlideClusterClientConfiguration</code> for additional options.
```

Comment:
Tags in regular comments aren't rendered. It is just a text.
Probably use backticks `` ` ``